tensorflow-op-codegen/src/main.rs

```diff
@@ -270,6 +524,19 @@ fn define_op<W: Write>(
             c_name: attr.name.clone(),
         });
     }
+    for output in op.output_arg.iter() {
+        let mut number_attr = output.number_attr.clone();
+        let mut number_attr_opt = None;
+        if !number_attr.is_empty() {
+            number_attr_opt = Some(number_attr.clone());
+        }
```

Comment:
These five lines can be done in one go:

```rust
let number_attr_opt = if output.number_attr.is_empty() {
  None
} else {
  Some(output.number_attr.clone())
};
```

which also reduces the amount of cloning.